automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -344,7 +342,7 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
 
   def randomClusterName: ClusterName = ClusterName(s"automation-test-a${makeRandomId().toLowerCase}z")
 
-  def defaultClusterRequest: ClusterRequest = ClusterRequest(Map("foo" -> makeRandomId()), enableWelder = Some(enableWelder))
+  val defaultClusterRequest: ClusterRequest = ClusterRequest(Map("foo" -> makeRandomId()), enableWelder = Some(enableWelder))
```

Comment:
`makeRandomId` is a side effect.. is this ok being a `val`? I'm wondering if we'll get naming conflicts if tests execute concurrently.

Reply:
oh, yea. Good call